eng/InstallRuntimes.proj

```diff
@@ -146,6 +146,21 @@
     <Copy SourceFiles="@(PrivateBuildFiles)" DestinationFolder="$(DotNetInstallDir.Replace('\\', '\'))" />
   </Target>
 
+<!--
+    Removes the private build registry keys
+-->
+
+  <Target Name="CleanupPrivateBuild"
```

Comment:
Maybe just `CleanupRegistry`. This doesn't remove the test runtime.

Reply:
I wanted to keep it more generic because maybe someday it could clean up the runtime itself.